docs/tutorials/working_with_the_ensemble.ipynb

```diff
@@ -19,7 +19,7 @@
   },
```

Comment:
<p>I'd like to have more details here. What happens to other frames after update (is object frame updated?)? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>I've added a new section, "Keeping the Object and Source Tables in Sync", which briefly goes through a brief example of a table being marked "dirty", the Ensemble being updated with that table, and an operation which triggers a sync.</p><p> </p><p>Any feedback there would be welcome!</p>